table.go

```diff
@@ -173,6 +183,16 @@ var TableList = []Table{
 		Model:               &messages.Message{},
 		NetworkVersionRange: AllNetWorkVersions,
 	},
+
+	// added to miner_info in nv17
+	{
+		Name:                "miner_beneficiary",
+		Schema:              1,
+		Task:                tasktype.MinerBeneficiary,
+		Model:               &miner.MinerBeneficiary{},
+		NetworkVersionRange: FromNetworkVersion17,
+	},
+
```

Comment:
@frrist use miner_beneficiary only from nv17

Reply:
Correct. The task will only produce data for network v17 (and any subsequent network upgrades). Technically, you could run it for all network versions, and it will produce an `OK` status processing report. Still, there will not be any data in its corresponding table until nv17.